classes/Risk.php

```diff
@@ -63,16 +71,25 @@ public function getFields()
     public function getTranslationsFieldsChild()
     {
         $this->validateFieldsLang();
-        return $this->getTranslationsFields(array('name'));
+
+        return $this->getFieldsLang();
```

Comment:
I've just removed it. There are no usages anymore in the core and modules should have stopped using it since 1.5